.github/workflows/ci.yml

```diff
@@ -7,8 +7,9 @@ jobs:
     runs-on: ubuntu-latest
     
     strategy:
+      fail-fast: false
       matrix:
-        ruby-version: [head, 3.0, 2.7, 2.6, 2.5, 2.4, 2.3]
+        ruby-version: [3.1, 3.0, 2.7, 2.6, 2.5]
```

Comment:
Is there no equivalent for `head` anymore? It would be nice if this didn't need to be updated manually on every minor version.

Reply:
From the commit message:
> For CI it doesn't make sense to test with future versions, since it
makes CI non reproducible. If we want to test with an upcoming Ruby
version we should do so in a scheduled job.

I'd find it annoying if CI breaks just because a new Ruby version got released which needs code changes on our side to fix.